.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: node_js
 dist: xenial
-node_js:
-  - "10"
```

Comment:
Validating before we merge:
- What versions of npm and node is Travis installing right now?
- What versions will it install after this change lands?

Reply:
Great questions. 
Status quo: 
```
node --version
v10.16.0
$ npm --version
6.9.0
$ nvm --version
0.34.0
```
We will install 10.15.3 (current version in the .nvmrc) over this via the setup script. 
The setup script will also always install the latest version of npm.

Reply:
why not add 10.15.3 to .nvmrc?

Reply:
I do this in a future PR: https://github.com/PolymerLabs/arcs/pull/3396/files

So far, .nvmrc uses 10.16.0

Reply:
I can change update nvmrc here, too.